### DISCUSSION TITLE
Welcome to apcsa-public Discussions!

### DISCUSSION THREAD

**Discussion by aspiece** [TEALSK12/apcsa-public#287] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by fmorriso** [comment]
Hello, my name is Fred Morrison, in my 8th year as a Microsoft TEALS volunteer, all within the Loudoun County Public School (LCPS) district (5 years at Potomac Falls High School, 3 at Stone Bridge High School). 

For the 2021-22 school year, I volunteer in  4 classes (1 AP CSA, 3 Computer Math - basically an Introduction to Java course), three of them on "A" days, the other one on "B" days.
